jazzer/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM openjdk:11
-
 COPY * /usr/bin/
-
-RUN mkdir -p /corpus
-
+RUN mkdir -p /corpus && \
+    apt-get update && \
+    rm /bin/sh && ln -s /bin/bash /bin/sh && \
+    apt-get -qq -y install curl unzip zip && \
+    curl -s https://get.sdkman.io | bash && \
+    chmod a+x "$HOME/.sdkman/bin/sdkman-init.sh"
+RUN source "$HOME/.sdkman/bin/sdkman-init.sh" && \
```

Comment:
is it possible `kotlin` is in a repository and can be fetched with a package manager?

separately: this step would add lots of extra layers to the default fuzzer image. if you're trying to fuzz kotlin, is it possible you can use a multi-stage docker build where in one of the images you build the kotlin target and in the other you just fuzz?

Reply:
ah okay no problem, I thought we wanted to combine everything into these fuzzer images, but that makes sense. Where would you like me to store the `kotlin` build image?

Current `fuzzme/kotlin-jazzer` Dockerfile looks like the following:

```FROM fuzzers/jazzer:0.9.1-openjdk11
RUN apt-get update && \
    rm /bin/sh && ln -s /bin/bash /bin/sh && \
    apt-get -qq -y install curl unzip zip && \
    curl -s https://get.sdkman.io | bash && \
    chmod a+x "$HOME/.sdkman/bin/sdkman-init.sh"
COPY fuzzme.kt .
RUN source "$HOME/.sdkman/bin/sdkman-init.sh" && \
    sdk install kotlin && \
    kotlinc fuzzme.kt -include-runtime -d fuzzme.jar

# Set to fuzz!
ENTRYPOINT []
CMD [ "/usr/bin/jazzer_driver", "--cp=/fuzzme.jar", "--target_class=fuzzme.FuzzmeKt" ]
```

Reply:
Looking to utilize the proposed multi-stage docker build:

```
FROM zenika/kotlin
COPY fuzzme.kt .
kotlinc fuzzme.kt -include-runtime -d fuzzme.jar

FROM fuzzers/jazzer
COPY --from=0 /fuzzme.jar .

# Set to fuzz!
ENTRYPOINT []
CMD [ "/usr/bin/jazzer_driver", "--cp=/fuzzme.jar", "--target_class=fuzzme.FuzzmeKt" ]
```

Reply:
that's beautiful